### ISSUE TITLE
Best solvent for DiI

### ISSUE THREAD

**Issue by ajuavinett** [cortex-lab/neuropixels#3]
Trying to determine what to dissolve the DiI in for the purposes of marking the electrode tract. Invitrogen sells one already dissolved in [ethanol](https://www.thermofisher.com/order/catalog/product/V22885) but it's unclear to me whether this is okay for the probes. In the IMEC manual, ethanol is *not* listed as an acceptable probe solvent, but isopropyl alcohol is.

Of course, you can also buy [DiI crystals](https://www.thermofisher.com/order/catalog/product/D282) and dissolve them yourself. Invitrogen [recommends](https://tools.thermofisher.com/content/sfs/manuals/mp00282.pdf) DMSO, DMF, or ethanol, but again, neither of these are explicitly approved by IMEC. In previous work, I've dissolved DiI in water (~10%) but it's really a pain to get it to dissolve.

So, what have people used, and have you noticed any problems with the probes afterwards?

Thanks in advance for your input!

**Comment by nsteinme** [comment]
Good question, I use the one already-dissolved in ethanol and it seems to have been working fine, perhaps because the ethanol dissolves within seconds. Might be worth asking IMEC whether we should be worried about that, though. In any case if anyone else has used something different I'd be happy to hear about it, too.

**Comment by nsteinme** [comment]
From Marleen Welkenhuysen at IMEC:

"We have done some tests in the past to assess the effect of cleaning methods on the impedance of the electrodes. They were done with the phase 1 probes but the electrode material (TiN) did not change since then. We noticed a 4-5x increase in impedance after overnight soaking in ethanol. However, rinsing with ethanol decreased the impedance slightly. The use of isopropylalcohol (IPA), on the other hand, did not change impedance after overnight soaking.
Nevertheless, I would not recommend using ethanol for toxicity reasons since the electrode material (TiN) is very porous and the ethanol might get trapped inside the pores and maybe it cannot evaporate in time before implantation. This is just a theory and we don’t have any data to back this up, but maybe this is something that needs to be verified?
In summary, I would try to dissolve the dye in IPA (not sure if this would work?) and wait long enough before implantation to allow sufficient evaporation of the solvent."

**Comment by weallen** [comment]
How long have you observed the DiI to stick around after doing the recording? I've noticed it will stay there for at least a few days, but I wondered if anyone has gone significantly longer.

**Comment by ajuavinett** [comment]
Hi @weallen, personally I've used it with Neuropixels ~2 weeks out, with no problem, and other people in my lab assure me they've done chronic rat recordings with Neuronexus probes and the DiI sticks around for  a month or even longer.
